router/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:bullseye-slim
 RUN apt-get update && apt-get install -y tini bash iptables network-manager radvd dnsmasq openvpn
+RUN update-alternatives --set iptables /usr/sbin/iptables-legacy; \
+    update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
```

Comment:
@kb2ma, @alexgg, FYI, I have found that, without this legacy selection, the ip(6)tables command executed in the container would not affect the host OS, as if having separate namespaces. The non-legacy version uses `nft` and I am yet to read about it and understand how it interacts with balenaOS.